assets/style.css

```diff
@@ -0,0 +1,251 @@
+  color: inherit;
+  margin-top: 1px;
+  background: linear-gradient(#888 50%, #777 100%);
+  border-radius: 23px;
+  width: 88px;
+  height: 23px;
+}
+
+.search-box-icon {
+  background-image: url("data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIGJhc2VQcm9maWxlPSJmdWxsIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB4bWxuczpldj0iaHR0cDovL3d3dy53My5vcmcvMjAwMS94bWwtZXZlbnRzIj4KPGcgZmlsbC1vcGFjaXR5PSIuNSI%2BCgk8cG9seWdvbiBwb2ludHM9IjkuMjA3LDcuMTI2IDcuNzkzLDguNTQxIDExLjc5MywxMi41NDEgMTMuMjA3LDExLjEyNiIgLz4KCTxwYXRoIGQ9Ik01LjkxNywzYzEuNjA4LDAsMi45MTcsMS4zMDgsMi45MTcsMi45MTdTNy41MjUsOC44MzMsNS45MTcsOC44MzNTMyw3LjUyNSwzLDUuOTE3UzQuMzA4LDMsNS45MTcsMyBNNS45MTcsMQoJCUMzLjIwMSwxLDEsMy4yMDEsMSw1LjkxN3MyLjIwMSw0LjkxNyw0LjkxNyw0LjkxN3M0LjkxNy0yLjIwMSw0LjkxNy00LjkxN0MxMC44MzMsMy4yMDEsOC42MzIsMSw1LjkxNywxTDUuOTE3LDF6IiAvPgo8L2c%2BCjxnPgoJPHBvbHlnb24gZmlsbD0iI2ZmZiIgcG9pbnRzPSI5LjIwNyw2LjEyNiA3Ljc5Myw3LjU0MSAxMS43OTMsMTEuNTQxIDEzLjIwNywxMC4xMjYiIC8%2BCgk8cGF0aCBmaWxsPSIjZmZmIiBkPSJNNS45MTcsMmMxLjYwOCwwLDIuOTE3LDEuMzA4LDIuOTE3LDIuOTE3UzcuNTI1LDcuODMzLDUuOTE3LDcuODMzUzMsNi41MjUsMyw0LjkxN1M0LjMwOCwyLDUuOTE3LDIKCQkgTTUuOTE3LDBDMy4yMDEsMCwxLDIuMjAxLDEsNC45MTdzMi4yMDEsNC45MTcsNC45MTcsNC45MTdzNC45MTctMi4yMDEsNC45MTctNC45MTdDMTAuODMzLDIuMjAxLDguNjMyLDAsNS45MTcsMEw1LjkxNywweiIgLz4KPC9nPgo8L3N2Zz4K");
+  width: 14px;
+  height: 14px;
+  vertical-align: middle;
+  float: left;
+}
+
+.search-box {
+  width: 72px;
+  height: 14px;
+  padding-left: 3px;
+  border: none;
+  background-color: transparent;
+  color: white;
+  float: right;
+}
+
+.search-box:focus {
+  outline: none;
+}
+
+.left-nav1 {
+  border-top-left-radius: 4px;
+  border-bottom-left-radius: 4px;
+}
+
+.right-nav1 {
+  width: 132px;
+  border-top-right-radius: 4px;
+  border-bottom-right-radius: 4px;
+  border-right: none;
+}
+
+.right-nav1:hover {
+  cursor: auto;
+  background: linear-gradient(#8b8b8b, #656565 50%, #787878 100%);
+}
+
+#content {
+  width: 100%;
+  height: 900px;
+}
+
+#mix {
+  width: 100%;
+  height: 600px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+#main-nav1 {
+  height: inherit;
+  background-image: url("https://web.archive.org/web/20140228114747im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg");
+  background-position: 0 600px;
+  background-size: cover;
+  width: 100%;
+}
+
+#texts1 {
+  width: 80%;
+  text-align: center;
+  margin-top: 100px;
+}
+
+#menu {
+  width: 70%;
+  padding-top: 100px;
+}
+
+#menu ul {
+  height: 40px;
+  border-radius: 10px;
+}
+
+#products {
+  width: 100%;
+  display: flex;
+}
+
+footer div {
+  margin: 10px;
+}
+
+.products {
+  width: 100%;
+  height: 200px;
+}
+
+footer {
+  width: 100%;
+  height: 80px;
+  display: flex;
+}
+
+.mate {
+  margin-top: 10px;
+}
+
+nav ul li {
+  display: inline-block;
+  margin-left: 10px;
+  padding-right: 10px;
+  border-right: 1px ridge gray;
+}
+
+.font-x {
+  font-size: 0.7em;
+  color: black;
```

Comment:
remove line #238 ( `color: black;`) as you are already using it in the class **.black-x**

Reply:
Thank you for everything, your suggestions were very accurate and exactly what I needed.